### DISCUSSION TITLE
keep-alive in node red log

### DISCUSSION THREAD

**Discussion by bzq-pl** [alexkn/node-red-contrib-homeconnect#74] [Q&A] [answered]
Hi,

is there any way to disable sending to node-red log "keep-alive" message? It's hard to search in log if it's full of those "keep-alive" messages ;-)

Regards
Piotr

**Comment by alexkn** [comment, marked as answer]
This is logged with `debug` log level.
You can set you node red log level to `info` or higher to suppress it.

Is there a reason why your log level is `debug`?

**Reply by bzq-pl** [reply]
you are right. i missed this setting. I turned it when I was doing some development and I forgot to move it info level.
